content/applications/marketing/events/create_events.rst

```diff
@@ -0,0 +1,427 @@
+
+.. note::
+   To create a new event registration product directly from this field, type in the name of the new
+   event registration product, and select either :guilabel:`Create` or
+   :guilabel:`Create and edit...` from the resulting drop-down.
+
+   Clicking :guilabel:`Create` creates the event registration product, which can be edited later.
+   Clicking :guilabel:`Create and edit...` creates the event registration product, and allows the
+   user to edit its configurations right away on a :guilabel:`Create Product` pop-up window.
+
+.. tip::
+   Existing event registration products can be modified directly from this field, as well, by
+   clicking the :guilabel:`➡️ (right arrow)` icon, located beside the event registration product.
+   Doing so reveals that product's form. Then, proceed to make any desired modifications.
+
+.. note::
+   In a database with the *Inventory* application installed, clicking the :guilabel:` ➡️ (right
+   arrow)` icon, located beside the event registration product reveals the product form inside the
+   *Inventory* application, with additional choices available to customize.
+
+   However, in a database *without* the *Inventory* application installed, clicking the
+   :guilabel:`➡️ (right arrow)` icon, located beside the event registration product reveals the
+   product form, but only with minimal fields and customizations visible.
+
+.. note::
+   Installing *just* the *Events* application on a trial database installs additional applications
+   automatically: :guilabel:`Discuss`, :guilabel:`Sales`, :guilabel:`Dashboards`,
+   :guilabel:`Invoicing`, and :guilabel:`Website`.
+
+Add or modify the cost of the ticket in the :guilabel:`Price` field.
+
+.. note::
+   It should be noted that a modified ticket price on the event form, in the :guilabel:`Tickets`
+   tab, does **not** change the price that appears on the ticket's product form. This modification
+   is specific to the event being configured.
+
+Next, determine a :guilabel:`Sales Start` and :guilabel:`Sales End` date in their respective fields.
+To do that, click into the blank field to reveal a calendar popover. From there, select the desired
+date and time, then click :icon:`fa-check` :guilabel:`Apply`.
+
+Then, if desired, designate a :guilabel:`Maximum` amount of that specific ticket that can be sold.
+
+The :guilabel:`Taken` column will populate with accurate data as tickets are sold.
+
+In the :guilabel:`Color` column, users have the option to add a custom color code to different
+tickets.
+
+To delete any tickets from the :guilabel:`Tickets` tab, click the :icon:`fa-trash-o`
+:guilabel:`(trash can)` icon on the corresponding line for the ticket that should be deleted.
+
+.. tip::
+   To add an optional :guilabel:`Description` column to the :guilabel:`Tickets` tab, click the
+   :guilabel:`additional options` :icon:`oi-settings-adjust` menu, located to the far-right of the
+   column titles.
```

Comment:
I think this formatting makes the most sense in this case– if the oi-settings-adjust is known as "additional options", that should be its label. WDYT?
```suggestion
   :icon:`oi-settings-adjust` :guilabel:`(additional options)` drop-down menu, located to the
   far-right of the column titles.
```